restclient/src/main/java/com/box/l10n/mojito/rest/client/AIServiceClient.java

```diff
@@ -0,0 +1,59 @@
+package com.box.l10n.mojito.rest.client;
+
+import com.box.l10n.mojito.rest.entity.OpenAICheckRequest;
+import com.box.l10n.mojito.rest.entity.OpenAICheckResponse;
+import com.box.l10n.mojito.rest.entity.OpenAIPrompt;
+import com.box.l10n.mojito.rest.entity.OpenAIPromptContextMessageCreateRequest;
+import com.box.l10n.mojito.rest.entity.OpenAIPromptCreateRequest;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AIServiceClient extends BaseClient {
+
+  static Logger logger = LoggerFactory.getLogger(AIServiceClient.class);
+
+  @Override
+  public String getEntityName() {
+    return "ai";
+  }
+
+  public OpenAICheckResponse executeAIChecks(OpenAICheckRequest openAICheckRequest) {
```

Comment:
Re naming: are OpenAICheckResponse & OpenAICheckRequest the right names for this class (i.e.: OpenAI-specific?)